ui/src/plugins/logger.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  install(Vue) {
+    function log(type, message, ...args) {
+      if (process.env.NODE_ENV === "production") {
+        return;
+      }
+      console[type](message, ...args);
+    }
+
+    Vue.prototype.$logger = {
```

Comment:
We need some unit tests for this. For example, they can test the message is correctly written or that the log level is correctly set depending on the function called.